devices.js

```diff
@@ -371,6 +371,20 @@ const livolo = {
 };
 
 const devices = [
+    // 3A Smart Home DE
+    {
+        zigbeeModel: ['LXN59-2S7LX1.0'],
+        model: 'LXN59-2S7LX1.0',
+        vendor: '3A Smart Home DE',
```

Comment:
https://aliexpress.ru/item/4001093548153.html shows this is a Zemismart device which doesn't not match the vendor here, what is the correct one?

Reply:
I bought it from exact link above, and it shows `"manufId":4478,"manufName":"3A Smart Home DE"` in database.db.

I have only 1-gang version btw.

Reply:
Also, the box was very generic, without any brand or logo on it.

Reply:
Thanks, added Zemismart as the whitelabe.